data/hai/hai reveal 1 intro.txt

```diff
@@ -309,15 +307,66 @@ mission "ZZ Event Hai Secret Leaks: FW"
 		event "news: hai video clip published"
 		event "pirates investigate leaks"
 		conversation "hai reveal free worlds news"
-	
-	
+
+
 event "pirates investigate leaks"
 	system "Waypoint"
 		government "Pirate"
 		add fleet "Marauder fleet IX" 2000
 		add fleet "Anti-Pirate Hai (No Wormhole Travel)" 700
 
 
+mission "Trader Directions to Hai Space"
+	landing
+	name "Investigate the Hai"
+	description "Follow the coordinates given to you by Captain Casey and meet the Hai."
+	source Sunracer
+	waypoint "Ultima Thule"
+	destination "Allhome"
+	to offer
+		or
+			has "ZZ Event Hai Secret Leaks: RS: offered"
+			has "ZZ Event Hai Secret Leaks: FW offered"
+	on offer
+		conversation
+			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that here, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Syndicate corporate machinery here on Sunracer, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`
+			`	The endless cycle of both the news and government propaganda being displayed on most screens in major thoroughfares is remarkably absent here, with only one of the screens on display covering the "violent alien race" at all - running stats on how many different versions of the story are now being told.`
+			`	Casting around the room you pick out a merchant who looks like they've been around the block a few times. In addition to being a woman who looks like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly. As you make your way over though you see her glare falter in a flash of recognition, before subsiding to a reserved scowl.`
```

Comment:
```suggestion
			`	Casting around the room, you pick out a merchant who looks like they've been around the block a few times. In addition to looking like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly; though as you make your way over, you see her glare falter in a flash of recognition before subsiding to a reserved scowl.`
```
What's happening here? Does the younger captain approach her and is sent away almost immediately? Not clear on that.

Reply:
Yeah they shouldn't be the same sentence cause they're two distinct events. You watch (from a distance) an exchange happen between her and the younger captain who is swiftly sent away. That moment resolves fully. Then you approach.

Reply:
```suggestion
			`	Casting around the room, you pick out a merchant who looks like they've been around the block a few times. In addition to looking like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly. After the captain is briskly sent away, you make you make your way over. Her glare falters in a flash of recognition before subsiding to a reserved scowl.`
```
How bout this?

Reply:
Oh now yes that does read well. The double of "you make" looks to be accidental but this structure is good.